xbmc/addons/gui/skin/SkinTimers.dox

```diff
@@ -133,8 +133,10 @@ Skin timers have the following available tags:
 | start         | An info bool expression that the skinning engine should use to automatically start the timer <b>(optional)</b>
 | reset         | An info bool expression that the skinning engine should use to automatically reset the timer <b>(optional)</b>
 | stop          | An info bool expression that the skinning engine should use to automatically stop the timer <b>(optional)</b>
-| onstart       | A builtin function that the skinning engine should execute when the timer is started <b>(optional)</b>
-| onstop        | A builtin function that the skinning engine should execute when the timer is stopped <b>(optional)</b>
+| onstart       | A builtin function that the skinning engine should execute when the timer is started <b>(optional)</b><b>can be repeated</b>
+| onstop        | A builtin function that the skinning engine should execute when the timer is stopped <b>(optional)</b><b>can be repeated</b>
```

Comment:
Does &lt;/b>&lt;b> insert a space between (optional) and "can be repeated"? I guess GH parses HTML so lets find out:

<b>(optional)</b><b>can be repeated</b>